### PR TITLE
[FSTORE-1346] Update doc dependencies

### PR DESCRIPTION
**Pull request by vatj**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -118,62 +118,62 @@ On docs.hopsworks.ai we implement the following versioning scheme:
...
 
-4. For this you can either checkout and make a local copy of the `upstream/gh-pages` branch, where
-`mike` maintains the current state of docs.hopsworks.ai, or just build documentation for the branch you are updating:
+4.  For this you can either checkout and make a local copy of the `upstream/gh-pages` branch, where
+    `mike` maintains the current state of docs.hopsworks.ai, or just build documentation for the branch you are updating:
 
-    Building *one* branch:
+        Building *one* branch:
 
-    Checkout your dev branch with modified docs:
```
Comment (robzor92) [inline CONTRIBUTING.md:126]: The section below here does not render correctly.

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -118,62 +118,62 @@ On docs.hopsworks.ai we implement the following versioning scheme:
...
 
-4. For this you can either checkout and make a local copy of the `upstream/gh-pages` branch, where
-`mike` maintains the current state of docs.hopsworks.ai, or just build documentation for the branch you are updating:
+4.  For this you can either checkout and make a local copy of the `upstream/gh-pages` branch, where
+    `mike` maintains the current state of docs.hopsworks.ai, or just build documentation for the branch you are updating:
 
-    Building *one* branch:
+        Building *one* branch:
 
-    Checkout your dev branch with modified docs:
```
Comment (robzor92) [inline docs/CONTRIBUTING.md:126]: The section below here does not render correctly.

**Review by robzor92 — CHANGES_REQUESTED** [review]

**Review by robzor92 — CHANGES_REQUESTED** [review]
The mkdocs-release workflow fails

**Review by robzor92 — CHANGES_REQUESTED** [review]
Is there no need to override this one also? https://github.com/logicalclocks/feature-store-api/pull/1275/files

**Comment by vatj** [comment]
Handled in another PR

**Pull request closed without merging**